trading_cost.py

```diff
@@ -0,0 +1,17 @@
+"""
+File of trading cost.
+"""
+
+def cost_currency(yesterday, today, price):
+    """
+    Calculate trading cost.
+
+    This function is designed to calculate "today's" trading cost.
+    "Today" will move by simulator, so we don't need to handle it
+    in this function.
+
+    The trading cost is calculated while selling the target.
```

Comment:
`target` is not defined here.
Maybe `The trading cost is accounted after selling.`